configurations/default/env.yml.tmp

```diff
@@ -29,3 +29,14 @@ MONGO_DB_NAME: otp_middleware
 OTP_API_ROOT: http://otp-server.example.com/otp
 OTP_PLAN_ENDPOINT: /plan
 MAXIMUM_PERMITTED_MONITORED_TRIPS: 5
+
+# SMS/email Notification configuration settings.
+NOTIFICATION_FROM_EMAIL: noreply@email.com
+NOTIFICATION_FROM_PHONE: +15551234
+# Twilio settings available at: https://twilio.com/user/account
+TWILIO_ACCOUNT_SID: your-account-sid
+TWILIO_AUTH_TOKEN: your-auth-token
```

Comment:
The SMS notification service variable needs to be added here too.

Reply:
To be addressed in #70.